spec/features/users_show_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Test the User show page' do
+  before :each do
+    @user = User.create(name: 'Jake', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Algeria.',
+                        posts_counter: 0)
+
+    # add five posts for the user
+    5.times do |i|
+      Post.create(author: @user, title: "Post #{i}", text: 'This is my new post', comments_counter: 0, likes_counter: 0)
+    end
+    visit user_path(@user.id)
+  end
+
+  it 'displays users profile picture' do
+    find('img[alt="Jake"]')
+  end

```

Comment:
- [ ] Please kindly assert using `expect` to confirm that the found profile picture is on the page